uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/shared/components/leave-calendar-month.component.spec.js

```diff
@@ -473,9 +473,9 @@
             });
           });
 
-          describe('when the balance change of the leave request is positive', function () {
```

Comment:
The TOIL `request_type` is a group, and any absence type which has a `allow_accruals_request` set to true, false into this category. So any request type with positive balance should already have `allow_accruals_request` set to true, so this will work too. cc @davialexandre